### PR TITLE
Addressed Buildifier lint issues and enabled Buildifier check in CI pipeline.

### LINKED ISSUE #126: Feature Request: Enable Bazel Starlark Linting and Formatting
## Tasks
- [x] Add Bazel target that can be used to check for lint and formatting issues using Buildifier
- [x] Add Bazel target that can fix lint and formatting issues using Buildifier.
- [ ] Update the existing codebase to address existing lint and formatting issues.
- [ ] Update the CI pipeline to execute the check target.

## Suggested Implementation
- First PR
  - Add [keith/buildifier-prebuilt](https://github.com/keith/buildifier-prebuilt) as a dev-only dependency.
  - Implement two runnable Bazel targets: one to check the lint issues and one to fix issues.
- Second PR
  - Fix all of the existing lint issues 
  - Add another entry to `bazel_commands` in the `buildbuddy.yml` file to execute the check Bazel target.

### PR DESCRIPTION
**Pull request by cgrindel**
Closes #126 

- Ran `//:buildifier.fix` on the codebase.
- Addressed issues uncovered by running `//:buildifier.check`.
- Enabled CI pipeline action to run `//:buildifier.check`.

### CODE REVIEW

File: test/fixtures/fixtures.bzl [hunk 1]
```diff
@@ -1,25 +1,15 @@
...
-    # "example/ios_app/p.xcodeproj" -> "test/fixtures/ios_app/project.xcodeproj"
-    return paths.join(
-        "test/fixtures",
-        xcodeproj.short_path.split("/")[1],
-        "project.xcodeproj",
-    )
-
 # Transition
 
+# buildifier: disable=unused-variable
```
Comment (brentleyjones) [inline test/fixtures/fixtures.bzl:12]: Can we instead use the underscore for this?

File: test/internal/build_settings/calculate_module_name_tests.bzl [hunk 2]
```diff
@@ -1,6 +1,8 @@
 """Tests for module name build setting functions."""
 
 load("@bazel_skylib//lib:unittest.bzl", "asserts", "unittest")
+
+# buildifier: disable=bzl-visibility
```
Comment (brentleyjones) [inline test/internal/build_settings/calculate_module_name_tests.bzl:5]: I see you opened an issue about this. I thought you could use internal in the same repo though 🤔

File: xcodeproj/internal/input_files_aspect.bzl [hunk 3]
```diff
@@ -124,6 +124,7 @@ def _input_files_aspect_impl(target, ctx):
     non_arc_srcs = []
     other = []
 
+    # buildifier: disable=uninitialized
```
Comment (brentleyjones) [inline xcodeproj/internal/input_files_aspect.bzl:127]: This one is a bug btw. If we rename the arguments it works. It's incorrectly looking at the variables declared later...

Not that we have to address that here, just commenting.

**Review by brentleyjones — APPROVED** [review]

File: test/internal/build_settings/calculate_module_name_tests.bzl (hunk 2 above)
Comment (cgrindel) [reply test/internal/build_settings/calculate_module_name_tests.bzl:5]: In my experience, it looks like you can use it one level up. In other other repos, I have worked through this by declaring them in the parent, in a separate file from the public APIs. Then, I declare a `bzl_library` target with the visibility that I want. It is something to consider. 🤷‍♂️

**Comment by cgrindel** [comment]
@brentleyjones It looks like I do not have permission to merge the PR. Could you do so or provide me with merge permission? Thanks.

**Comment by cgrindel** [comment]
Sorry. I spaced out. I just saw your other comments. Let me take a look at them.

File: xcodeproj/internal/input_files_aspect.bzl (hunk 3 above)
Comment (cgrindel) [reply xcodeproj/internal/input_files_aspect.bzl:127]: Ah. I wasn't sure. I have not used embedded functions in Starlark.

File: test/fixtures/fixtures.bzl (hunk 1 above)
Comment (cgrindel) [reply test/fixtures/fixtures.bzl:12]: Sorry. I don't follow. Do you mean the warning category? I am basing them off of [this list](https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md).

**Comment by brentleyjones** [comment]
@cgrindel And you should have permission to merge now.

File: test/fixtures/fixtures.bzl (hunk 1 above)
Comment (brentleyjones) [reply test/fixtures/fixtures.bzl:12]: We can rename the unused variables to have a leading underscore instead of ignoring the warning.

Comment (cgrindel) [reply test/fixtures/fixtures.bzl]: I think you meant the variable names. 🙂 That does work. I removed the directive and added the underscores.

**Pull request merged**